### PR TITLE
Updated EntityTypeEnum and EntityType descriptions

### PR DESCRIPTION
**Pull request by rachel-nml**

### CODE REVIEW

File: website/static/downloads/swagger.yaml [hunk 1]
```diff
@@ -2219,28 +2231,28 @@ components:
...
+        * 0x433D (17213 decimal) - Metadata Namespace Transaction.
         * 0x414D (16717 decimal) - Mosaic Definition Transaction.
         * 0x424D (16973 decimal) - Mosaic Supply Change Transaction.
         * 0x414E (16718 decimal) - Register Namespace Transaction.
         * 0x424E (16974 decimal) - Address Alias Transaction.
         * 0x434E (17230 decimal) - Mosaic Alias Transaction.
         * 0x4154 (16724 decimal) - Transfer Transaction.
         * 0x4155 (16725 decimal) - Modify Multisig Account Transaction.
-        * 0x4141 (16705 decimal) - Aggregate Complete Transaction.
+        * 0x4141 (16705 decimal) - Aggregate Completed Transaction.
```
Comment (shinneng) [inline website/static/downloads/swagger.yaml:2245]: @rachel-nml it should be reverted to Aggregate Complete Transaction

File: website/static/downloads/swagger.yaml [hunk 2]
```diff
@@ -2219,28 +2231,28 @@ components:
...
         * 0x4241 (16961 decimal) - Aggregate Bonded Transaction.
         * 0x4148 (16712 decimal) - Hash Lock Transaction.
-        * 0x4150 (16720 decimal) - Account Properties Address Transaction.
-        * 0x4250 (16976 decimal) - Account Properties Mosaic Transaction.
-        * 0x4350 (17232 decimal) - Account Properties Entity Type Transaction.
+        * 0x4150 (16720 decimal) - Account Property Address Transaction.
+        * 0x4250 (16976 decimal) - Account Property Mosaic Transaction.
+        * 0x4350 (17232 decimal) - Account Property Entity Type Transaction.
```
Comment (shinneng) [inline website/static/downloads/swagger.yaml:2250]: https://github.com/proximax-storage/go-xpx-chain-sdk/blob/master/api.md#type-accountproperties
It should be Account Properties

File: website/static/downloads/swagger.yaml [hunk 3]
```diff
@@ -2219,28 +2231,28 @@ components:
       description: |
         The entity type:
         * 0x4158 (16728 decimal) - Blockchain Upgrade Transaction.
-        * 0x4159 (16729 decimal) - Network Config Transaction.
-        * 0x413D (16701 decimal) - Address Metadata Transaction.
-        * 0x423D (16957 decimal) - Mosaic Metadata Transaction.
-        * 0x433D (17213 decimal) - Namespace Metadata Transaction.
+        * 0x4159 (16729 decimal) - Network Config Entity Type Transaction.
```
Comment (shinneng) [inline website/static/downloads/swagger.yaml:2234]: I think we can exclude "Entity Type"

**Comment by rachel-nml** [comment]
I have made another pull request to revise those transactions you have pointed out. Thank you.

**Pull request merged**